container/conductor/visibility.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+# This is a copy of /container/visibility.py copied for accessibility inside
+# the conductor
+from __future__ import absolute_import
+
+import inspect
+import logging
+import sys
+
+from ruamel import ordereddict
+
+from structlog import wrap_logger
+from structlog.processors import JSONRenderer
+from structlog.dev import ConsoleRenderer
+from structlog.stdlib import filter_by_level
+from structlog.processors import format_exc_info, TimeStamper
+
+logging.basicConfig(
+    level=logging.INFO,
+    stream=sys.stderr,
+    format='%(message)s',
+)
+
+def local_var_info(logger, call_name, event_dict):
+    if logger.getEffectiveLevel() > logging.DEBUG or call_name != 'debug':
+        return event_dict
+    caller = inspect.stack()[3]
+    event_dict.update({
+        'locals': caller[0].f_locals,
+    })
+    return event_dict
+
+def _unroll_odict(od):
```

Comment:
Removed in favor of json.dumps (during debug mode) and standard encoding during othe modes.